doc/nrf/ug_thingy53.rst

```diff
@@ -0,0 +1,258 @@
+.. _ug_thingy53:
+
+Working with Thingy:53
+######################
+
+.. contents::
+   :local:
+   :depth: 2
+
+Nordic Thingy:53 is a battery-operated prototyping platform for IoT Systems.
+
+Thingy:53 integrates the following components:
+
+* nRF5340 SoC - Supporting Bluetooth® Low Energy, 802.14.5 based protocols and Near Field Communication (NFC).
+* nRF21540 RF FEM - Range extender with integrated power amplifier (PA)/low-noise amplifier (LNA).
+* nPM1100 PMIC - Power Management IC with integrated dual-mode buck regulator and battery charger.
+
+For more information on the product, see the `Thingy:53 product page`_ and the `Nordic Thingy:53 User Guide`_.
+The |NCS| provides support for developing applications on the Thingy:53.
+
+.. _thingy53_serialports:
+
+Connecting to Thingy:53 serial ports
+************************************
+
+For connecting to Thingy:53, use a serial terminal.
+
+Thingy:53 uses the following UART baud rate configuration:
+
+.. list-table::
+   :header-rows: 1
+   :align: center
+
+   * - UART Interface
+     - Baud Rate
+   * - UART
+     - XXXXXXX
+
+
+Firmware
+********
+
+Nordic Thingy:53 comes preprogrammed with the Edge Impulse firmware, which allows the Thingy:53 to connect to Edge Impulse Studio over Bluetooth LE using the nRF Edge Impulse mobile application or through USB on your PC.
+The firmware allows you to capture sensor data, which in turn you can be use to train and test machine learning models in Edge Impulse Studio.
```

Comment:
Typeo: The firmware allows you to capture sensor data, which in turn you can be **used** to train...

Reply:
which in turn you can use to train...
or
which in turn can be used to train...
:)